pages/linux/screenkey.md

```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display your keys.
```

Comment:
What do you think about this, @ashishbinu?
```suggestion
> A screencast tool to display keys currently being pressed.
```

Reply:
How about this?

```suggestion
> A screencast tool to display the keys pressed.
```

Reply:
Sounds better than mine, except would sound even bit better if you removed `the`, like this:
```suggestion
> A screencast tool to display keys pressed.
```

Reply:
I don't know how much value we want to give this, but I think `the keys pressed` may be a bit easier to read for non-native speakers.

Reply:
Well, it just doesn't sound very good, and I can't exactly put my finger on it, but the subject "keys pressed" seems a bit too small for `the` 🤔